### PR TITLE
Adds Atmos magboots, and tweaks magboot sprites.

### PR DESCRIPTION
**Pull request by kugamo**
## What Does This PR Do
Adds a new type of magboot for atmospheric technicians, it would replace the generic magboots in the atmos suit storage units. Atmospheric Magboots uniquely prevent unsafe unwrenchings from occurring when toggled on, this functionality has been extended to the Advanced Magboots as well. In all other ways atmospheric magboots are just a reskin of normal magboots. 
(normal magboots already protect from space wind moving you when toggled on)

In addition to the addition of atmospheric magboots, this PR alters existing on character magboot sprites  for normal magboots and Advanced magboots to reflect their magpulse state. The old sprites were ambiguous, some had on sprites and no off sprites, this PR adds off lights and makes on lights consistent. 

## Why It's Good For The Game
Having a way for atmos techs to work seemlessly with high pressure pipes is good, also I believe sub-department specific gear is fun and these new boots will expand upon sub-department identity. For the sprite alterations, clear visual feedback about the state of your items adds to ease of play.

## Images of changes
**New Atmos Magboots**
dreamseeker_0k8EdmDcd2

0HWeIpuaGP

**On-Character sprite changes**
qTVVUO3q3V
xH8TdRRbqc
1Ko5DdhBGH

## Changelog
:cl:
add: Adds Atmos Magboots.
tweak: Tweaks existing magboot on-character sprites.
/:cl:

### CODE REVIEW

**Comment by taukausanake** [comment]
I could have sworn there were Atmos Magboots but they were fluff items? How strange, I know I've seen the sprites for them before.

So does this mean that normal magboots don't protect you from pipe gusts? I kind of thought when they were powered on your feet are stuck no matter what, passive magnetism rather than reactive magnetism. Aside from stuns of course

**Comment by AffectedArc07** [comment]
These *were* someones fluff item 
image

However they arent actually in the game anymore
image

**Comment by kugamo** [comment]
So I discovered that the fluff item was added in 2013 by @SniperYeti here: https://github.com/ParadiseSS13/Paradise/commit/1c3d72e471f405236a732998bcba8e1b734b39ca

Those didnt have any new functionality, and were just a reskin. I have attempted to contact this person. I do note that the sprites in this PR are not the original sprites for atmos magboots, I made new ones based off the generic magboots.

No normal magboots do not protect against unsafe unwrenchings.

**Comment by kenken244** [comment]
I don't see why atmos magboots should have any more functionality than regular magboots. if their purpose is to prevent you getting thrown around, then make it so dangeour pipes don't affect anyone with ANY magboots, still, having magboots match the hardsuit is cool.

File: code/ATMOSPHERICS/atmospherics.dm [hunk 1]
```diff
@@ -190,9 +191,14 @@ GLOBAL_DATUM_INIT(pipe_icon_manager, /datum/pipe_icon_manager, new())
...
 
+			for(var/obj/item/clothing/shoes/magboots/usermagboots in user.get_equipped_items())
+				if(usermagboots.gustprotection && usermagboots.magpulse)
+					safefromgusts = TRUE
+
 			//You unwrenched a pipe full of pressure? let's splat you into the wall silly.
 			if(unsafe_wrenching)
-				unsafe_pressure_release(user,internal_pressure)
+				if(safefromgusts) to_chat(user, "<span class='italics'>Your magboots cling to the floor as a great burst of wind bellows against you.")
+				else unsafe_pressure_release(user,internal_pressure)
```
Comment (SteelSlayer) [inline code/ATMOSPHERICS/atmospherics.dm:201]: The preferred style for ifs is that you put the code that gets ran when it succeeds, on a new line, like this:
```suggestion
				if(safefromgusts) 
					to_chat(user, "<span class='italics'>Your magboots cling to the floor as a great burst of wind bellows against you.</span>")
				else 
					unsafe_pressure_release(user,internal_pressure)
```
Also, you are missing a `</span>` at the end of your `to_chat` (included in the suggestion).

Comment (kugamo) [reply code/ATMOSPHERICS/atmospherics.dm]: I only recently learned why I shouldn't code like this. thank you for the suggestion, I will commit this.

**Review by variableundefined — APPROVED** [review]
Seems OK.

**Pull request merged**